modules/introduction/pages/welcome.adoc

```diff
@@ -10,36 +10,36 @@ Hello there!
+The Internet Computer is the third major blockchain innovation — a blockchain that scales smart contract computation, runs them at web speed, processes and stores data efficiently, and provides powerful software frameworks to developers. By making this possible, the Internet Computer enables the complete reimagination of how systems and apps operate.
 
 Interested in learning more? Check out the following resources:
 
 * link:https://www.youtube.com/watch?v=YWHTNr8RZHg&list=PLuhDt1vhGcrf4DgKZecU3ar_RA1cB0vUT&index=17&ab_channel=DFINITY[Primer to the Internet Computer], a high-level overview of the Internet Computer in under 10 minutes
 * link:https://medium.com/dfinity[The Internet Computer Review], our blog covering updates for the Internet Computer 
-* link:https://thereboot.com/[The Reboot], our tech publication exploring the issues with the current internet
+* link:https://thereboot.com/[The Reboot], our tech publication exploring issues with the current internet
 
 Follow us on the official link:https://twitter.com/dfinity[DFINITY Twitter] for the latest updates.
 
 
 [[for-developers]]
 == For: Developers
-The Internet Computer hosts special smart contracts, called link:https://medium.com/dfinity/software-canisters-an-evolution-of-smart-contracts-internet-computer-f1f92f1bfffb[canisters]. A canister is a bundle of link:https://webassembly.org/[WebAssembly] bytecode logic and memory pages. Apps on the Internet Computer are created from one or more canisters. 
+The Internet Computer hosts special smart contracts, called link:https://medium.com/dfinity/software-canisters-an-evolution-of-smart-contracts-internet-computer-f1f92f1bfffb[canisters]. A canister smart contract is a bundle of link:https://webassembly.org/[WebAssembly] bytecode logic and memory pages. Dapps on the Internet Computer are created from one or more canisters. 
 
-Canisters can serve web content directly to end-users, and users can interact with blockchain services without holding tokens. When building on the Internet Computer, you can deploy code directly to the Internet. 
+Canister smart contracts can serve web content directly to end-users, and users can interact with blockchain services without holding tokens. When building on the Internet Computer, you can deploy code directly to the Internet. 
 
-The Internet Computer can eventually host any number of canisters (i.e., code & state). Moreover, it can run them concurrently with unbounded on-chain capacity. This means you can create dapps that scale. 
+The Internet Computer can eventually host any number of canister smart contractss (i.e., code & state). Moreover, it can run them concurrently with unbounded on-chain capacity. This means you can create dapps that scale. 
```

Comment:
```suggestion
The Internet Computer blockchain can eventually host any number of canister smart contracts (i.e., code & state). Moreover, it can run them concurrently with unbounded on-chain capacity. This means you can create dapps that scale. 
```